docs/en/03_Upgrading/index.md

```diff
@@ -35,6 +36,14 @@ composer require silverstripe/asset-admin ^1.0
 
 This will also install the `graphql` module for GraphQL API access to your SilverStripe system, which powers the `asset-admin` module.
 
+## Migrate to dotenv
+
+SilverStripe 4 requires the use of `.env` and "real" environment variables instead of `_ss_environment.php` for your environment configuration.
+
+You'll need to move your constants to a new `.env` file before SilverStripe will build successfully.
+
+If you are using SilverStripe Platform you will need to use a temporary adapter to import the constants that SilverStripe Platform sets in `_ss_environment.php` into environment variables. You can use [this example file](https://gist.github.com/robbieaverill/74fbfff6f438c94f6325107e4d7b2a45) and include it at the top of your `mysite/_config.php` file until SilverStripe Platform supports `.env` generation.
```

Comment:
It's not specifically about SilverStripe Platform, and that caveat will be out of date shortly.

Keep it general: "if you're not able to move your webserver away from using _ss_environment.php files"